web/password_prompt.js

```diff
@@ -33,12 +33,12 @@ import { PasswordResponses } from './pdfjs';
 /**
  * @class
```

Comment:
I'm curious if we still need this comment, now that the thing below is an actual proper Class?